build.gradle

```diff
@@ -18,6 +18,7 @@ targetCompatibility = 1.6
 repositories {
     maven { url "http://oss.sonatype.org/content/repositories/snapshots" }
     maven { url "http://repo.maven.apache.org/maven2" }
+    mavenLocal();
```

Comment:
The ```mavenLocal()``` method was added after a discussion for this custom release.  This does not need to be in this PR but is the a reason why it does not currently exist?